packages/workflow/src/records/handler/unassign.ts

```diff
@@ -17,6 +17,39 @@ import { toUnassigned } from '@workflow/records/state-transitions'
 import { indexBundleToRoute } from '@workflow/records/search'
 import { sendBundleToHearth } from '@workflow/records/fhir'
 import { auditEvent } from '@workflow/records/audit'
+import {
+  Extension,
+  findExtension,
+  isTaskOrTaskHistory,
+  Resource,
+  resourceIdentifierToUUID,
+  Task
+} from '@opencrvs/commons/types'
+import { getTokenPayload } from '@opencrvs/commons/authentication'
+import { getUserOrSystemByCriteria } from '@workflow/records/user'
+import { getTaskBusinessStatus } from '@workflow/features/task/fhir/utils'
+
+function sortDescending(
+  a: Resource & { lastModified: string },
+  b: Resource & { lastModified: string }
+) {
+  return new Date(b.lastModified).valueOf() - new Date(a.lastModified).valueOf()
+}
```

Comment:
https://github.com/opencrvs/opencrvs-core/blob/develop/packages/commons/src/fhir/task.ts#L203 We actually have a helper for this already

Reply:
Yeah i saw. Those were in gateway. I suppose we can move them into commons.

Reply:
wasn't it in commons already? I think the new one is more versatile and better tbh, but keeping the amount of helper functions lesser can help keep our code consistent and, well often less code = less bugs

Reply:
Oh sorry! I missed the link. I'll update it as such.